connector/google/google.go

```diff
@@ -252,34 +263,95 @@ func (c *googleConnector) createIdentity(ctx context.Context, identity connector
 	return identity, nil
 }
 
+func (c *googleConnector) getAllGroups(ctx context.Context, userKey string) ([]string, error) {
+	directMemberships, err := c.adminSrv.Groups.List().
+		UserKey(userKey).
+		Domain(c.domain).
+		Context(ctx).
```

Comment:
The pagination token is missing here. Is it ok? Previously it was always specified.

Reply:
It was actually not handled at all in the original either. Pushed a change which fetches all pages.